res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -4,9 +4,16 @@
   <id>org.mixxx.Mixxx</id>
   <launchable type="desktop-id">mixxx.desktop</launchable>
   <metadata_license>CC-BY-SA-3.0</metadata_license>
-  <project_license>GPL-2.0</project_license>
+  <project_license>GPL-2.0-or-later</project_license>
   <name>Mixxx DJ Software</name>
   <summary>Everything you need to perform live DJ mixes</summary>
+  <!-- Must be explicitly specified as not requiring any content rating. -->
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="2.3.0" date="2021-06-28">
```

Comment:
Please add an entry to the release checklist for updating this file.

Including the actual release date causes more work, because we have to add it right before tagging the release.

Reply:
Is this `<releases>` tag actually required? We should try to avoid it.

Let's say we want to release 2.3.1. This means we need to publish the release announcement before actually tagging the release here, otherwise we can't add the link to this file.

Reply:
Yes, `releases` is a requirement. The date is not a requirement though. I think we should copy the changelog here rather than use the URL to the announcement. That way we can update this file before tagging.

Reply:
Then let's omit the date.

The `<releases>` tag is not required, at least not when packaging for a regular Linux distribution. Version information is usually defined sparately by the packaging and not inline. If ever possible I would like to omit it entirely.

Reply:
Maybe this is desired: https://github.com/Be-ing/mixxx/pull/57